src/calculateRentalCost.js

```diff
@@ -28,7 +28,7 @@
  */
 
 function calculateRentalCost(days) {
```

Comment:
The function calculateRentalCost does not have a return statement. Make sure to calculate the rental cost based on the given days and return the result.